core-bundle/src/Security/Authentication/ContaoLoginAuthenticationListener.php

```diff
@@ -48,7 +48,7 @@ protected function requiresAuthentication(Request $request): bool
     {
         return $request->isMethod('POST')
             && $request->request->has('FORM_SUBMIT')
-            && 0 === strncmp($request->request->get('FORM_SUBMIT'), 'tl_login', 8);
+            && preg_match('/^tl_login(_[0-9]+)?$/', $request->request->get('FORM_SUBMIT'), $matches);
```

Comment:
```suggestion
            && preg_match('/^tl_login(_[0-9]+)?$/', $request->request->get('FORM_SUBMIT'));
```

You are never using the `$matches` variable, are you?